### ISSUE TITLE
Add support to Heat Detector

### ISSUE THREAD

**Issue by Bhargav-Rao** [SOBotics/Open-Reports#5]
Hydrant has an endpoint to return the posts which have no feedback.

https://api.higgs.sobotics.org/Reviewer/PendingReviews?dashboardName=Hydrant&PageNumber=1&PageSize=1

Calling the API would return something like:

     {
         pageNumber: 1,
         pageSize: 1,
         totalPages: 3309,
         data: [
             3575
         ]
    }

**Comment by BaumMitAugen** [comment]
For my own information: The data is the ID in the report URL
    https://higgs.sobotics.org/Hydrant/report/[ID]

I'll implement this on Friday.

@Bhargav-Rao given the number of unhandled reports, do we need some restriction on the batch size of would Higgs deal with someone trying to open 3k reports at once in a sensible way? SO just IP-bans you when you overdo it (which happens somewhere in the region of 100 reports, depending on your USs).

**Comment by Bhargav-Rao** [comment]
Yeah, let's restrict it at 100 reports per batch. That would be better.
